client/components/domains/domain-suggestion/style.scss

```diff
@@ -47,6 +47,11 @@
 		.is-placeholder & {
 			color: transparent;
 		}
+
+		.flag {
```

Comment:
We should avoid generic html selectors for specificity. `.domain-suggestion__content .flag {}` would be the preferred way to modify `.flag` here.